tensorflow_model_optimization/g3doc/guide/clustering/clustering_comprehensive_guide.ipynb

```diff
@@ -279,6 +279,25 @@
         "clustered_model.summary()"
       ]
     },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Cluster convolutional layers per channel\n",
+        "\n",
+        "The clustered model could be passed to further optimizations such as a [post training quantization](https://www.tensorflow.org/lite/performance/post_training_quantization). If the quantization is done per channel, then the model should be clustered per channel as well. This increases the accuracy of the clustered and quantized model\n",
```

Comment:
Minor:

-> such as [post training quantization]

Missing period at end of last sentence.